ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -962,11 +962,11 @@ tests:
         MIRROR_IMAGES=true
         INSTALLER_PROXY=true
         NETWORK_TYPE=OVNKubernetes
-      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi'
+      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @disconnected'
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
```

Comment:
The work flow is for connected (will run step enable-qe-catalogsource), you may need to add one workflow cucushift-installer-rehearse-baremetalds-ipi-ovn-disconnected (call step enable-qe-catalogsource-disconnected)

Reply:
cc @jianzhangbjz

Reply:
yes,  make sense. good catching.

Reply:
updated, thanks